x-pack/plugins/apm/public/components/alerting/chart_preview/index.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  AnnotationDomainTypes,
+  Axis,
+  BarSeries,
+  Chart,
+  LineAnnotation,
+  niceTimeFormatter,
+  Position,
+  RectAnnotation,
+  RectAnnotationDatum,
+  ScaleType,
+  Settings,
+  TickFormatter,
+} from '@elastic/charts';
+import { EuiSpacer } from '@elastic/eui';
+import { max as getMax, min as getMin } from 'lodash';
+import moment from 'moment';
+import React from 'react';
+import styled from 'styled-components';
+import { Coordinate } from '../../../../typings/timeseries';
+import { useTheme } from '../../../hooks/use_theme';
+
+const ChartWithHeight = styled(Chart)`
+  height: 150px;
+`;
+
+interface ChartPreviewProps {
+  yTickFormat?: TickFormatter;
+  data?: Coordinate[];
+  threshold: number;
+}
+
+export function ChartPreview({
+  data = [],
+  yTickFormat,
+  threshold,
+}: ChartPreviewProps) {
+  const theme = useTheme();
+  const thresholdOpacity = 0.3;
+  const timestamps = data.map((d) => d.x);
+  const xMin = moment.utc(getMax(timestamps)).valueOf();
```

Comment:
For consistency let's use `Math.max`

```suggestion
  const xMin = moment.utc(Math.max(timestamps)).valueOf();
```

(I assume that's what `getMax` does?)